Lombiq.NodeJs.Extensions/build/NodeJsExtensionsNpmPackage.targets

```diff
@@ -4,7 +4,7 @@
   <Target Name="EnablePnpm">
     <!-- Enable the latest version of pnpm@8 using explicit version number. -->
     <!-- Ignore sporadic errors when this is run by multiple threads at the same time. -->
-    <Exec Command="corepack enable &amp;&amp; corepack prepare pnpm@8.6.8 --activate"
+    <Exec Command="corepack enable &amp;&amp; corepack prepare pnpm@8.6.9 --activate"
```

Comment:
Why not use 8.6.12, the latest version?

Reply:
8.6.9 was the latest last month when I updated that :/

Reply:
Just FYI, once you force users to use at least v16.17 of Node.js, you'll be able to use the [dist-tag](https://docs.npmjs.com/cli/v9/commands/npm-dist-tag) `latest-8` to cover all patch and minor version updates, as mentioned here: https://www.npmjs.com/package/pnpm?activeTab=versions ([docs](https://github.com/nodejs/node/pull/44646), [more docs](https://github.com/nodejs/node/pull/44646)).

Reply:
Thanks for the remark! Not having open version selectors lets us avoid surprise breakage like https://github.com/Lombiq/Orchard-Base-Theme/issues/84 though.